tests/Unit/AddCustomProviderTest.php

```diff
@@ -25,9 +25,7 @@ public function tearDown()
     public function testIfApiProviderOnAuthWasSetCorrectly()
```

Comment:
If you change the return of the method `invalidRequest($provider)` to `true` this test will fail. Because the provider `companies` isn't defined. To it works you should set a config like `config(['auth.guards.company.provider' => 'companies']);` before execute the `AddCustomProvider::handle()` method.

Reply:
Just change the method `AddCustomProvider::invalidRequest` to return `true` after check if provider exists. When you run the tests this test should fails. Because the provider `companies` doesn't exists on config. Do you understood?

Reply:
Yeah! going to fix that.